WordPressAuthenticator/Authenticator/WordPressAuthenticatorStyles.swift

```diff
@@ -75,32 +85,14 @@ public struct WordPressAuthenticatorStyle {
         self.primaryTitleColor = primaryTitleColor
         self.secondaryTitleColor = secondaryTitleColor
         self.disabledTitleColor = disabledTitleColor
+        self.textButton = textButton
+        self.textButtonHighlight = textButtonHighlight
```

Comment:
sorry didn't comment on this earlier, a non-blocking nit: maybe `buttonTextColor`/`buttonTextHighlightedColor` would sound more clear that these two are colors not buttons? also more consistent with the names of other arguments (same for the property names)

Reply:
I agree, these should be suffixed with `Color`. 

This whole struct has gotten a bit big for my tastes, but I'll leave a more thorough refactor for another time 😉